src/rpdk/core/contract/resource_client.py

```diff
@@ -402,7 +402,6 @@ def _call(self, payload):
             for key in [
                 "callbackContext",
                 "action",
-                "requestData",
                 "region",
```

Comment:
request data contains all the requested properties if we removing it then this logging isnt really helpful